project-base/storefront/components/Layout/Header/MenuIconic/MenuIconicItemUserAuthenticated.tsx

```diff
@@ -16,50 +23,104 @@ export const MenuIconicItemUserAuthenticated: FC = () => {
         ['/customer', '/customer/orders', '/customer/edit-profile'],
         url,
     );
+    const [isClicked, setIsClicked] = useState(false);
+    const [isHovered, setIsHovered] = useState(false);
```

Comment:
I would suggest adding a delay when the user hovers over the icon, similar to how `const isMenuOpenedDelayed = useDebounce(isMenuOpened, 200);` is implemented in `NavigationItem.tsx`

Reply:
used useDebounce, that pushed me to add check if hover is used on desktop or not. There were ability to hover with mouse on developer respo tool. Thx